### DISCUSSION TITLE
Multiple Startup Projects (Maui Single Project)

### DISCUSSION THREAD

**Discussion by epsmae** [dotnet/maui#19773] [Q&A]
I am wondering if it is not possible to start multiple platform at the same time in Visual Studio 2022 with Maui Single Project.
For Example, I would like to start and Debug the Windows and Android App in parallel when I hit F5.
The default maui project template sets `<SingleProject>true</SingleProject>`

In Configure Startup projects I can only select each projects once, so no way to use windows and android.

https://learn.microsoft.com/en-us/visualstudio/ide/how-to-set-multiple-startup-projects?view=vs-2022 

Is there another way to do that?

**Comment by nbaulesglobalsolutions** [comment]
do we need an android device setup for this.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Manuel ***@***.***>
Sent: Tuesday, January 9, 2024 4:04:13 PM
To: dotnet/maui ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dotnet/maui] Multiple Startup Projects (Maui Single Project) (Discussion #19773)


I am wondering if it is not possible to start multiple platform at the same time in Visual Studio 2022 with Maui Single Project.
For Example, I would like to start and Debug the Windows and Android App in parallel when I hit F5.
The default maui project template sets <SingleProject>true</SingleProject>

In Configure Startup projects I can only select each projects once, so no way to use windows and android.

https://learn.microsoft.com/en-us/visualstudio/ide/how-to-set-multiple-startup-projects?view=vs-2022

Is there another way to do that?

**Reply by epsmae** [reply]
@nbaulesglobalsolutions no not really you can choose any two platforms with physical device or simulator.
